src/Schema/Exception/InvalidSchemaException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAPIValidation\Schema\Exception;
+
+use Exception;
+
+class InvalidSchemaException extends Exception
```

Comment:
This file is not used anywhere. Is it there for upcoming changes?

Reply:
Here is another issue. Doctrine coding standard won't let you name exception as "Exception". You need to remove "Exception" suffix in order to pass the quality checks.

Reply:
I am going to update CI config to run these checks automatically for each new PR

Reply:
Nah, we just removed it's only usage in one of "too defencive checks". Will clean it up

Reply:
Removed